pkg/assisted/agentclusterinstall.go

```diff
@@ -638,12 +638,14 @@ func (builder *AgentClusterInstallBuilder) Delete() error {
 		builder.Definition.Name, builder.Definition.Namespace)
 
 	if !builder.Exists() {
-		return fmt.Errorf("agentclusterinstall cannot be deleted because it does not exist")
```

Comment:
Could you add a glog statement here please? I remember this came up in an earlier PR and I think the conclusion was to add them going forward

Reply:
Done!  And I adjusted all of the PRs accordingly.  I forgot about adding the glog.

Reply:
Thanks!